packages/nimble-components/src/wafer-map/tests/matrix-renderer.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { WaferMap } from '..';
+import { MatrixRenderer } from '../modules/matrix-renderer';
+
+/*
+ * Sometimes methods from render worker become bugged after being built, bundled and copied to a different location
+ * with these tests we ensure they behave as expected after this process
+ */
```

Comment:
I don't think this comment adds much value so I'd suggest deleting it. Tests generally serve to validate build and execution so describing ways they can fail seems redundant to me.

Also, it sounds like the build process is buggy? Can we get to the bottom of why that is? Is that going to cause problems on the pipeline or for developers running local builds? We want our infrastructure to be extremely reliable.

Reply:
Indeed the comment was not very useful, I deleted it. 
By 'render worker become bugged after being built...' I wanted to say that making changes to render-worker.ts or rollup config can break the code without being noticed, @munteannatan had some incidents, but the potential of breaking a code portion while developing is expected and should be prevented with unit tests, now that I thought a second time my comment didn't make much sense. Beside that the pipeline is stable and doesn't fail without a reason intermittently, from that perspective we should not worry.